lib/vector-core/src/event/vrl_target.rs

```diff
@@ -52,18 +52,22 @@ pub struct TargetIter<T> {
     _marker: PhantomData<T>,
 }
 
+fn create_log_event(metadata: EventMetadata, value: Value) -> LogEvent {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ Looking at the other `match` arms below passing `value` into the function first would match up better.